src/Symfony/Bridge/Twig/Tests/Extension/WorkflowExtensionTest.php

```diff
@@ -88,4 +101,17 @@ public function testGetMarkedPlaces()
         $this->assertSame(array('ordered', 'waiting_for_payment'), $this->extension->getMarkedPlaces($subject));
         $this->assertSame($subject->marking, $this->extension->getMarkedPlaces($subject, false));
     }
+
+    public function testGetMetadata()
+    {
+        if (!class_exists(InMemoryMetadataStore::class)) {
+            $this->markTestSkipped('This test requires symfony/component:4.1.');
```

Comment:
symfony/component does not exist :) You probably mean `symfony/workflow` here, right?